src/tabs/docs/readme.md

```diff
@@ -8,6 +8,10 @@ Foundation version of the tabs directive.
  	_(Defaults: false)_ :
  	Whether tabs appear vertically stacked.
 
+ * `open-first-tab`
+  _(Defaults: false)_ :
```

Comment:
This actually defaults to true. If it's absent, it's explicitly set to true: https://github.com/pineconellc/angular-foundation/pull/196/files#diff-9073c3d0209295459ab79e243f181520R16